app/src/tingapp.js

```diff
@@ -14,7 +14,13 @@ class Tingapp {
 
         const newDocumentPath = path.join(tempDir, 'untitled.tingapp');
         console.log(newDocumentPath);
-        fsextra.copySync('./default.tingapp', newDocumentPath);
+        // check for dev to select where to fetch resources from
+        if(/[\\/]electron-prebuilt[\\/]/.test(process.execPath)){
+          fsextra.copySync('./app/default.tingapp', newDocumentPath);
+        }else{
+          fsextra.copySync(path.join(process.resourcesPath,'app.asar','default.tingapp'), newDocumentPath);
+        }
```

Comment:
Yes, that would require the change to electron 1.x.x which shouldn't be a problem, when I actually wrote this 1.0.0 hadn't been released